### ISSUE TITLE
(Linux) нет иконки приложения

### ISSUE THREAD

**Issue by smotim** [amnezia-vpn/amnezia-client#80]
На fedora 36 после установки обновления 2.0.10 появился ярлык в меню, но у него нет значка. Попробовал через редактор меню руками поставить значок из ...client/share/icons, но ярлык так и остался пустым. Может значок как-то не подходит?
![изображение](https://user-images.githubusercontent.com/57951811/178599692-e4972cfd-adff-49ef-8ea3-305f8de4f9e6.png)

**Comment by smotim** [comment]
UPD: немного поторопился. После перезагрузки Gnome иконка появилась. Т.е. выбор через редактор помогает решить проблему

**Comment by zlocate** [comment]
По-умолчанию иконка также отсутствует в XFCE (Manjaro).

**Comment by Skipper92** [comment]
> На fedora 36 после установки обновления 2.0.10 появился ярлык в меню, но у него нет значка. Попробовал через редактор меню руками поставить значок из ...client/share/icons, но ярлык так и остался пустым. Может значок как-то не подходит? ![изображение](https://user-images.githubusercontent.com/57951811/178599692-e4972cfd-adff-49ef-8ea3-305f8de4f9e6.png)

Приветствую. Прошу прощения за такой вопрос, но как Вы установили Amnezia Installer на Fedora?

**Comment by smotim** [comment]
> > На fedora 36 после установки обновления 2.0.10 появился ярлык в меню, но у него нет значка. Попробовал через редактор меню руками поставить значок из ...client/share/icons, но ярлык так и остался пустым. Может значок как-то не подходит? ![изображение](https://user-images.githubusercontent.com/57951811/178599692-e4972cfd-adff-49ef-8ea3-305f8de4f9e6.png)
> 
> Приветствую. Прошу прощения за такой вопрос, но как Вы установили Amnezia Installer на Fedora?

Скачал из релизов, сделал исполняемым и запустил. Когда я первый раз ставил, там еще папку установки пришлось искать потом. Сейчас вроде ярлык сам создается. Если что, редактор меню в помощь. У вас какие-то сложности с установкой?

**Comment by Skipper92** [comment]
> > > На fedora 36 после установки обновления 2.0.10 появился ярлык в меню, но у него нет значка. Попробовал через редактор меню руками поставить значок из ...client/share/icons, но ярлык так и остался пустым. Может значок как-то не подходит? ![изображение](https://user-images.githubusercontent.com/57951811/178599692-e4972cfd-adff-49ef-8ea3-305f8de4f9e6.png)
> > 
> > 
> > Приветствую. Прошу прощения за такой вопрос, но как Вы установили Amnezia Installer на Fedora?
> 
> Скачал из релизов, сделал исполняемым и запустил. Когда я первый раз ставил, там еще папку установки пришлось искать потом. Сейчас вроде ярлык сам создается. Если что, редактор меню в помощь. У вас какие-то сложности с установкой?
> 
> UPD: конкретно как впн-приложение работает плохо. Не все протоколы включаются, а какие включаются, работают через раз. Может не хватает каких-то зависимостей. Я пользуюсь только настройками, конфиги испортирую в network manager или другие приложения

Да, проблемы есть, но скорее из-за того, что моё знакомство с Linux только началось (пару дней)=) Спасибо большое за ответ)

**Comment by CherepanovAV** [comment]
В текущей версии ярлык на рабочем столе создается, но в нем указана картинка из каталога /usr/share/pixmaps/  
А такой картинки там не лежит. При установке забыта скопироваться

**Comment by pokamest** [comment]
Please check if it fixed in 4.1.0 - https://github.com/amnezia-vpn/amnezia-client/releases/tag/4.1.0.1
